org.musicbrainz.Picard.json

```diff
@@ -159,30 +154,16 @@
             ]
         },
         {
-            "name": "python-pkgconfig",
+            "name": "python-discid",
```

Comment:
I'd prefer we keep using python-libdiscid instead:

* python-discid uses ctypes, which in my experience is very brittle;
* python-libdiscid uses Cython to compile the module, which probably makes it faster; (I have not measured though, but the fragility of ctypes is enough anyway)

Reply:
2 reasons : 

1. upstream use `python-discid`
2. Build failed with python 3.7 on my machine.

Reply:
Those are 2 good points. Let's use python-discid then.

Reply:
Please do that in its own commit though, separated from the move to org.kde.Platform//5.12, explaining the above.